.mailmap

```diff
@@ -1780,3 +1780,5 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+XLR8 <trungpqhe171493@gmail.com>
+XLR8 <86862725+AcceleratorHTH@users.noreply.github.com>
```

Comment:
Do you want `XLR8` to be the name recorded in the AUTHORS file?

Either way there should not be two lines with different name/email combinations here:
https://github.com/sympy/sympy/blob/3b255faf11bcaf19b267fed9cc7e0c8da81ff6dc/.mailmap#L129-L148